docs/control-structures.rst

```diff
@@ -18,12 +18,17 @@ Functions are executable units of code within a contract. Functions may only be
 
 Functions may be called internally or externally depending on their :ref:`visibility <function-visibility>`. Functions may accept input arguments and return variables in order to pass values between them.
 
-.. _function-visibility:
-
 Visibility
 ----------
 
-All functions must include exactly one visibility decorator.
+.. _function-visibility:
+
+You can optionally declare a function's visibility by using a :ref:`decorator <function-decorators>`. There are three visibility levels in vyper:
+
+    * **External**: exposed in the selector table, can be called by an external call into this contract
+    * **Internal**: can be invoked only from within this contract. not available to external callers
```

Comment:
```suggestion
    * ``@internal``: can be invoked only from within this contract. Not available to external callers
```

Please add also here that the default visibility is `@internal` now (i.e. if no visibility modifier is set). (I know it's later stated as well but still useful info here)